src/_pytest/fixtures.py

```diff
@@ -1124,13 +1124,25 @@ def getfixtureinfo(self, node, func, cls, funcargs=True):
             argnames = getfuncargnames(func, cls=cls)
         else:
             argnames = ()
+
+        # Collecting parametrize arguments so they are ignored later
+        parametrize_argnames = []
+        for marker in node.iter_markers(name="parametrize"):
+            from _pytest.mark import ParameterSet
+
+            p_argnames, _ = ParameterSet._parse_parametrize_args(
```

Comment:
I think it makes sense to move this inside the `if` below

Reply:
Indeed!